modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,254 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-node-affinity.adoc
+// * nodes/scheduling/nodes-scheduler-pod-affinity.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+
+ifeval::["{context}" == "nodes-scheduler-pod-affinity"]
+:pod:
+endif::[]
+ifeval::["{context}" == "nodes-scheduler-node-affinity"]
+:node:
+endif::[]
+ifeval::["{context}" == "olm-adding-operators-to-a-cluster"]
+:olm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="olm-overriding-operator-pod-affinity_{context}"]
+
+ifdef::olm[]
+= Controlling where an Operator is installed
+endif::olm[]
+
+ifdef::pod[]
+= Using pod anti-affinity to control where an Operator is installed
+endif::pod[]
+
+ifdef::node[]
+= Using node affinity to control where an Operator is installed
+endif::node[]
+
+By default, when you install an Operator, {product-title} installs the Operator pod to one of your worker nodes randomly. However, there might be situations where you want that pod scheduled on a specific node or set of nodes.
+
+For example, some Operators require being scheduled on a node with a particular architecture, such as linux/amd64, linux/arm64, or Windows.
+
+Or you might want Operators that work closely together to be scheduled on the same host, or on hosts located on the same rack. Alternatively, you might want your Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues.
```

Comment:
```suggestion
By default, when you install an Operator, {product-title} installs the Operator pod to one of your worker nodes randomly. However, there might be situations where you want that pod scheduled on a specific node or set of nodes.

The following examples describe situations where you might want to schedule an Operator pod to a specific node or set of nodes:

* If an Operator requires a particular architecture, such as linux/amd64, linux/arm64, or Windows
* If you want Operators that work together scheduled on the same host or on hosts located on the same rack
* If you want Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues
```
I keep thinking this could benefit from being a list. WDYT?

Reply:
Also, our guidance is a little unclear as to how to refer to architectures. The SSG only has guidance for [AMD64](https://redhat-documentation.github.io/supplementary-style-guide/#amd-64). Based on a conversation with @kelbrown20, I think what you have here is consistent with the OCP docs that refer to architectures.

Reply:
Maybe breaking up the operating system and architecture aspects in the list?

* If an Operator requires a particular architecture, such as `amd64` or `arm64`
* If an Operator requires a particular arhitecture, such as Linux or Windows